README.rst

```diff
@@ -2,7 +2,7 @@
 graylog-formula
 ================
 
-A saltstack formula that can be used to manage graylog installations on RHEL based systems using a package or tar file.
+A saltstack formula that can be used to manage graylog installations on RHEL nd Debian based systems using a package or tar file.
```

Comment:
I see a typo it's "RHEL and Debian based"

Reply:
Thanks for pointing that out, it's fixed.